### ISSUE TITLE
[!!][Project Plan] Silver roles

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-roles-responsbilities#9]
> Define roles to be included (as opposed, or in conjunction with roles defined in Silver),

Why "as opposed"?

Don't we want to start with the work that Silver has already done -- even if we eventually suggest different?!?

**Comment by orvalle** [comment]
We would prefer to use those. It would only be "opposed" if the AG roles are not already defined. We don't want a dependency on that project for our work. If they're not ready we will go ahead with our own.

**Comment by shawna-slh** [comment]
I _think_ this is the latest: [Job Stories for Stakeholders](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Job_Stories_for_Stakeholders)
Would be good to check with Silver folks to confirm.

In any case, I think can delete "as opposed, or" :-)

**Comment by dboudreau** [comment]
@slhenry - We are familiar with the document, as it is something that Bill had brought to us in the beginning. We decided to create our own because in its current format, that information does not help us get to where we need to go. Their roles are not defined, and that has an impact when it comes to breaking down roles and responsibilities. It can help more broadly to understand the landscape of stakeholders, but we're diving much more deeper than that in our work. And we're focused on the SDLC specifically.

The link you gave us points to job stories for stakeholders. What we need is definitions for those roles and both are different. Its the key distinction that prevents us form fully aligning to it.

"As opposed to" is from me, and I'll blame language barriers. We did not mean of it in opposition to what we're doing, but as something we can work with and align with. That said, the roles in that document don't really fit with what we're up to much.

We can align to names whenever and wherever it works (when it's specific enough), but that list goes way beyond what we're trying to talk about. Some of the names can work, but as long as we don't have definitions for those roles in that document, we can't really use them. It doesn't mean that over time, we won't be able to align and harmonize to another W3C document such as this one.

That W3C document would need to define the 30+ roles so we all get in agreement as to what they mean. Because if you can't tell exactly what that role does, how can you define what it needs to do. As an example, a UI designer has a different role than a UX designer, and both play a very different role in the project lifecycle. Labeling both roles as designers actually blurs the lines, so that's not helping our goal. 

Since we need to move forward and this is foundational to what we're doing, we can't really wait for AG to define those roles, but if they ever do, we'll be happy to align.

**Comment by dboudreau** [comment]
**Notes:**
- Use job stories in our work
- Feed Silver with other stories
- Etc.

**Comment by dboudreau** [comment]
Assuming we were all good with the reasoning, here. Hence, closing.

**Comment by dboudreau** [comment]
Based on today's call, I suggest closing this issue. We will coordinate with the Silver taskforce to find some way to harmonize if possible, but our scope and goals being different than theirs, we see very little value in aligning at this time. Please close the issue if you agree.

**Comment by shawna-slh** [comment]
1. In [Requirements doc](https://www.w3.org/WAI/EO/wiki/RA11y_Matrix), still need to change
> Define roles to be included (as opposed, or in conjunction with roles defined in Silver),

How about:
> Define roles to be included (considering [roles defined in Silver](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Job_Stories_for_Stakeholders))

2. I think we agreed to check in with Silver folks and let them know what we're working on. (Please CC public-eo-archive@w3.org :)

I think close this issue only after that contact has been made, so we don't forget. :-)

**Comment by dboudreau** [comment]
Change made.

**Comment by dboudreau** [comment]
@slhenry - Who are the Silver folks, besides Jeanne Spellman? Who should we add to this email?

**Comment by shawna-slh** [comment]
Jeanne Spellman <jspellman@paciellogroup.com>
Shawn Lauriat <lauriat@google.com>
AGWG Chairs <group-ag-chairs@w3.org>

And they can decide if & how to share the info with others working on Silver. :-)

Thanks.

**Comment by dboudreau** [comment]
Sweet. Thank you.

**Comment by jspellman** [comment]
The Job Stories are not meant to be roles.  We developed them for usability testing in the Silver Design Sprint.  We don't have consensus yet on roles.  Several people are resisting roles-based design for Silver because of the complexity of roles and how difficult it can be to decide which defined role applies to an individual.  I am being convinced by the arguments, even though I used to be very pro-roles.  Large companies can have tens of different design job titles, many of which cross over into what I would call "developer" role.  Small companies have one person who who is designer, developer and tester.  They would miss key information if they entered Silver as a Developer.  It's tricky, and we don't have consensus on the approach yet.
